views/home.hbs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Firefox Breach Alerts</title>
+        <link rel="stylesheet" href="/css/foundation.min.css">
```

Comment:
Should we put the CSS into /css/vendor/ directory to match the JS code?
For example:
- public/css/app.css
- public/css/vendor/foundation.min.css